code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -24,6 +25,31 @@
 	icon_state = "gag"
 	w_class = 1
 
+/obj/item/clothing/mask/muzzle/tapegag
+	name = "tape gag"
+	desc = "MHPMHHH!"
+	icon_state = "tapegag"
+	item_state = null
+	w_class = 1
+	resist_time = 200
+	species_fit = list("Unathi", "Tajaran", "Vulpkanin")
+	sprite_sheets = list(
+		"Unathi" = 'icons/mob/species/unathi/mask.dmi',
+		"Tajaran" = 'icons/mob/species/tajaran/mask.dmi',
+		"Vulpkanin" = 'icons/mob/species/vulpkanin/mask.dmi'
+		)
+
+/obj/item/clothing/mask/muzzle/tapegag/proc/tear_off(mob/living/carbon/human/user) //snowflaek, called in /mob/living/carbon/human/unEquip() in mob/living/carbon/human/inventory.dm
+	var/atom/movable/R = new /obj/item/trash/tapetrash
+	if(user.species.bodyflags & HAS_FUR)

```

Comment:
We already have the ability to adjust fire damage on the species datum level--we don't need a flag specifically for it.